### PR TITLE
Redis message broker

### PR DESCRIPTION
**Pull request by thedrow**

### CODE REVIEW

**Comment by dobosevych** [comment]
@auvipy it seems that the build didn't work well. Can you restart it?

**Comment by auvipy** [comment]
Could not parse version constraint: >=3.7,

  at /opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/site-packages/poetry/core/semver/helpers.py:144 in parse_single_constraint
      140│         return version
      141│ 
      142│     from poetry.core.semver.exceptions import ParseConstraintError
      143│ 
    → 144│     raise ParseConstraintError(f"Could not parse version constraint: {constraint}")
      145│ 
nox > Command poetry lock failed with exit code 1
nox > Session test-3.7(suite='functional') failed.

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [build-system]
 build-backend = "poetry.core.masonry.api"
-requires = ["poetry-core>=1.0.0"]
+requires = ["poetry-core>=1.1.0a7"]
```
Comment (thedrow) [inline pyproject.toml:3]: Why do we need a7?

File: pytest_celery/compat.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+try:
+    from functools import cached_property
+except ImportError:
+    # TODO: Remove this backport once we drop Python 3.7 support
+    from cached_property import cached_property
+
+cached_property = cached_property
```
Comment (thedrow) [inline pytest_celery/compat.py:7]: This statement has no effect. cached_property is already exported as a symbol.
```suggestion
```

**Review by auvipy — APPROVED** [review]

**Pull request merged**